tests/copy-verbose.earth

```diff
@@ -1,5 +1,5 @@
 VERSION --use-copy-include-patterns 0.5
```

Comment:
should we update the version here or did you intentionally skip for now?

Reply:
good eyes! There's a few tests in place specifically for old functionality, but I only ever grepped for `0.6` and clearly missed a few old tests.